packages/pcd/pod-pcd/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": "@pcd/tsconfig/ts-library.json",
+  "compilerOptions": {
+    "lib": [
+      "dom"
+    ],
```

Comment:
is dom necessary here?

Reply:
Same as elsewhere - came from the template.  Is this DOM as in Document Object Model?